Applications/Spire/Source/Ui/RealSpinBoxModel.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Spire/Spire/RealSpinBoxModel.hpp"
+
+using namespace Nexus;
+using namespace Spire;
+
+namespace {
+  auto to_real(const QVariant& value) {
+    static auto locale = [] {
+      auto locale = QLocale();
+      locale.setNumberOptions(locale.numberOptions().setFlag(
+        QLocale::OmitGroupSeparator, true));
+      return locale;
+    }();
+    return CustomVariantItemDelegate().displayText(value,
+      locale).toStdString().c_str();
```

Comment:
This function is called ```to_real``` but it returns a ```const char*```. Apart from being undefined behavior, the name of the function is not reflective of what this function actually does.

Reply:
Regarding the name, how is this function different from the other to_real(...) functions? Is it just semantics because the return type is const char*?

Reply:
A function named ```to_X``` should return an object of type ```X```.